datalake/processing_stack.py

```diff
@@ -15,6 +15,9 @@
 )
 from aws_cdk.core import Tags
 
+CHECK_METADATA_CONTENT_JOB_DEFINITION_PREFIX = "check_metadata_content"
```

Comment:
What's the purpose of using this constant `CHECK_METADATA_CONTENT_JOB_DEFINITION_PREFIX` instead of task name as it is used for all other tasks ?

Reply:
It was necessary to make the name available to a test.

Reply:
Removed test and inlined constant as per our discussion.